compiler/rustc_ast_lowering/src/lib.rs

```diff
@@ -2495,14 +2495,27 @@ impl<'hir> GenericArgsCtor<'hir> {
 
         let id = lcx.next_node_id();
         let hir_id = lcx.next_id();
+
+        let constness = match constness {
```

Comment:
We already know that `constness` is `Yes` from a few lines above.

Reply:
Yep, fixed.